### PR TITLE
🌐 Add Japanese translation for `docs/ja/docs/tutorial/encoder.md`

### PR DESCRIPTION
**Pull request by SwftAlpc**
This PR translates the tutorial/encoder.md to Japanese.
Please review tutorial/encoder.md.

~Issue #1571 generates a new directory to support Japanese.
PR #1656 generates a new directory for tutorial.
I will fix dependency after #1571 & #1656 are merged.~

### CODE REVIEW

File: docs/ja/docs/tutorial/encoder.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+例えば、データベースに保存する必要がある場合です。
+
+そのために、**FastAPI** は`jsonable_encoder()`関数を提供しています。
+
+## `jsonable_encoder`の使用
+
+JSON互換のデータのみを受信するデータベース`fase_db`があるとしましょう。
+
+例えば、`datetime`オブジェクトはJSONと互換性がないので受け取りません。
```
Comment (tokusumi) [inline docs/ja/docs/tutorial/encoder.md:13]: 例えば、`datetime`オブジェクトはJSONと互換性がないので、このデータベースには受け取られません。

**Review by tokusumi — APPROVED** [review]
I approved your translation.
I add a not important comment. please refer it if you want

File: docs/ja/docs/tutorial/encoder.md (hunk 1 above)
Comment (SwftAlpc) [reply docs/ja/docs/tutorial/encoder.md]: I adopt this one because it's a clearer Japanese translation!

**Comment by SwftAlpc** [comment]
@tokusumi 
Thanks for the review!
May I ask for confirmation?

**Review by tokusumi — APPROVED** [review]
Great! Thank you.

File: docs/ja/docs/tutorial/encoder.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+データ型（Pydanticモデルのような）をJSONと互換性のあるもの（`dict`や`list`など）に変更する必要がある場合があります。
+
+例えば、データベースに保存する必要がある場合です。
+
+そのために、**FastAPI** は`jsonable_encoder()`関数を提供しています。
+
+## `jsonable_encoder`の使用
+
+JSON互換のデータのみを受信するデータベース`fase_db`があるとしましょう。
```
Comment (delhi09) [inline docs/ja/docs/tutorial/encoder.md:11]: "fake_db" in original sentence.
(Probably its's typo and I know its's a mere example db name.)

File: docs/ja/docs/tutorial/encoder.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+例えば、`datetime`オブジェクトはJSONと互換性がないので、このデーターベースには受け取られません。
```
Comment (delhi09) [inline docs/ja/docs/tutorial/encoder.md:13]: It's a little better to modify "データーベース" to "データベース" because elsewhere it is referred to as "データベース".

**Review by delhi09 — COMMENTED** [review]
It looks mostly good.
I pointed out some details.

**Comment by tiangolo** [comment]
📝 Docs preview for commit 5f4ea5f78530730b98c519ba62959fa3457b3d2d at: https://6499ca883df3740528860850--fastapi.netlify.app

**Comment by alejsdev** [comment]
Thank you! @SwftAlpc 🤓 

And thanks to everyone else for the reviews 🧐🔍

**Pull request merged**